morebits.js

```diff
@@ -456,6 +456,9 @@ Morebits.quickForm.element.prototype.compute = function QuickFormElementCompute(
 			if (data.disabled) {
 				subnode.setAttribute('disabled', 'disabled');
 			}
+			if (data.required) {
+				subnode.setAttribute('required', true);
```

Comment:
```suggestion
				subnode.setAttribute('required', 'required');
```
Or `subnode.setAttribute('required', '');` or `subnode.required = true`. Setting the value to true [is invalid in HTML5](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes).

Reply:
Right you are, thanks!